### DISCUSSION TITLE
Pending Open Channels Not Broadcasted/Already Spent

### DISCUSSION THREAD

**Discussion by Batka0** [lightningnetwork/lnd#8238] [Troubleshooting]
I have a similar problem to https://github.com/lightningnetwork/lnd/issues/7505 due to high fees and improperly set fees given the mempool environment.

I have one channel open (e51b6bfb0f4f424e431f53dab45a85fd2912f4c32796b3737bf9a8b048e12b8f:1) that did not open for over 3 weeks and the tx is not found in my nodes local mempool (1.5gb) or block explorers. I attempted to force close it thinking perhaps that would be useful but it has not changed the state of the channel.
My node views the tx as invalid and already spent. I'm not sure if this is because the channel creation was connected to a utxo that was never confirmed or how the process started.

I also have a 2nd additional channel pending open (47d982dbf940fc374b3e98d81d03520a703254334eb38a4f2d65a1f6b91259cf:0) that has not yet expired but is also showing the same error messages.

Similar to the github issue, when I attempted to release the output it tells me [lncli] rpc error: code = Unknown desc = unknown output and attempting the psbt fund command also resulted in [lncli] rpc error: code = Unknown desc = input 0 not found in list of non-locked UTXO

Unlike in the github issue, neither appears in listunspent. Could you point me in the direction for some next step troubleshooting?

**Comment by guggero** [comment]
It's possible that because the transaction was removed from the mempool at some point, your node has spent the inputs to those funding transactions already for something else and they will never be able to confirm.
To confirm that, can you please do the following for each channel:
 - Run `lncli listchaintxns` and locate the transaction of each channel open transaction (the first part before the colon of the channel point is the TXID)
 - Look at the inputs to the channel funding transaction and look them up on chain with a block explorer
 - If one or more of the inputs has already been spent (it must be confirmed to be safe), the channel will never become operational and you can use `lncli abandonchannel` to get rid of it (that's a dangerous command, please read the `--help` output carefully and only use it after making sure about the inputs being spent)

**Reply by Batka0** [reply]
Neither e51b6bfb0f4f424e431f53dab45a85fd2912f4c32796b3737bf9a8b048e12b8f nor 47d982dbf940fc374b3e98d81d03520a703254334eb38a4f2d65a1f6b91259cf show in listchaintxns.

**Reply by guggero** [reply]
Can you restart your node and check the logs for attempted transaction broadcasts?

**Reply by guggero** [reply]
Thanks. So I checked the two funding transactions e51b6bfb0f4f424e431f53dab45a85fd2912f4c32796b3737bf9a8b048e12b8f and 47d982dbf940fc374b3e98d81d03520a703254334eb38a4f2d65a1f6b91259cf, and their inputs have all been spent. So the channels will never be operational and you can use the abandonchannel mentioned above (read the help for the required parameter that I won't mention here to avoid anyone copy/pasting it for the wrong situation).

**Reply by Batka0** [reply]
Thank you very much for that confirmation! I'll move forward with the abandonchannel (and the special flag!). As always, appreciate your swift assistance.
